packages/marshal/src/marshal-justin.js

```diff
@@ -396,7 +396,7 @@ const decodeToJustin = (encoding, shouldIndent = false) => {
         return out.close(']');
       }
     } else {
-      const names = ownKeys(rawTree);
+      const names = /** @type {string[]} */ (ownKeys(rawTree));
```

Comment:
Aye, I checked. `rawTree` is an `Encoding` which is a serializable JSON structure with only `string` keys. The typing of `ownKeys` is not generic, so it can't pick that subtlety. I can leave a comment to that effect.